### ISSUE TITLE
Find references

### ISSUE THREAD

**Issue by withered-magic** [withered-magic/starpls#125]
General find references functionality hasn't been implemented yet

**Comment by hauserx** [comment]
Quick question: is the idea to parse all .bzl files in the repo?

**Comment by withered-magic** [comment]
Yep, initial functionality will probably only work across the current workspace (and maybe any external repo files that you currently have open)

Initial implementation will probably look similar to this: https://rust-analyzer.github.io/blog/2019/11/13/find-usages.html
